### PR TITLE
Make build fail

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

File: script/build [hunk 1]
```diff
@@ -14,26 +21,28 @@ if [ -n "$CF_KV_TOKEN" ]; then
...
 
+# Export to files
+jq -n "$integration_details" --argjson integration_details "$integration_details" > "$integration_details_path"
+jq -n "$data" --argjson data "$data" > "$data_path"
 
-if [ ! -d node_modules ]; then
-  script/bootstrap
-fi
+
+script/bootstrap
```
Comment (balloob) [inline script/build:45]: Will this mess with local builds

**Review by balloob — APPROVED** [review]

File: script/build [hunk 2]
```diff
@@ -14,26 +21,28 @@ if [ -n "$CF_KV_TOKEN" ]; then
...
 
 else
-  curl -sSLf https://dev--home-assistant-analytics.netlify.app/data.json | jq . > ./site/public/data.json
-  curl -sSLf https://rc.home-assistant.io/integrations.json | jq . > ./site/public/integration_details.json
+  data=$(curl -sSLf https://dev--home-assistant-analytics.netlify.app/data.json)
+  integration_details=$(curl -sSLf https://rc.home-assistant.io/integrations.json)
 fi
 
+# Export to files
+jq -n "$integration_details" --argjson integration_details "$integration_details" > "$integration_details_path"
```
Comment (balloob) [inline script/build:41]: Weird we need to pass data twice. Btw what if the Json contains double quotes, that won’t interfere with the double quotes that wrap the variable?

**Review by balloob — APPROVED** [review]

File: script/build (hunk 1 above)
Comment (ludeeus) [reply script/build:45]: It just runs yarn install, shouldn't be a problem

File: script/build (hunk 2 above)
Comment (ludeeus) [reply script/build:41]: Can do `echo "$var" | jq . > path` if you prefer

**Pull request merged**